data/README.md

```diff
@@ -1,337 +1,2 @@
-| [TSV](tsv/vie_latn_hanoi_phonetic.tsv) | vie | Vietnamese | Vietnamese | Latin | Hà Nội | False | Phonetic | True | 15,240 |
-| [TSV](tsv/vie_latn_hanoi_phonetic_filtered.tsv) | vie | Vietnamese | Vietnamese | Latin |  | True | Phonetic | True | 15,240 |
-| [TSV](tsv/vie_latn_hue_phonetic_filtered.tsv) | vie | Vietnamese | Vietnamese | Latin |  | True | Phonetic | True | 15,236 |
-| [TSV](tsv/vie_latn_hcmc_phonetic_filtered.tsv) | vie | Vietnamese | Vietnamese | Latin |  | True | Phonetic | True | 14,537 |
-| [TSV](tsv/vie_latn_hcmc_phonetic.tsv) | vie | Vietnamese | Vietnamese | Latin | Hồ Chí Minh City | False | Phonetic | True | 15,237 |
-| [TSV](tsv/vol_latn_phonemic.tsv) | vol | Volapük | Volapük | Latin |  | False | Phonemic | True | 363 |
-| [TSV](tsv/vol_latn_phonetic.tsv) | vol | Volapük | Volapük | Latin |  | False | Phonetic | True | 565 |
-| [TSV](tsv/wau_latn_phonemic.tsv) | wau | Waurá | Wauja | Latin |  | False | Phonemic | True | 152 |
-| [TSV](tsv/wel_latn_nw_phonemic.tsv) | wel | Welsh | Welsh | Latin | North Wales | False | Phonemic | True | 8,530 |
-| [TSV](tsv/wel_latn_nw_phonemic_filtered.tsv) | wel | Welsh | Welsh | Latin |  | True | Phonemic | True | 8,473 |
-| [TSV](tsv/wel_latn_sw_phonemic.tsv) | wel | Welsh | Welsh | Latin | South Wales | False | Phonemic | True | 9,999 |
-| [TSV](tsv/wel_latn_sw_phonemic_filtered.tsv) | wel | Welsh | Welsh | Latin |  | True | Phonemic | True | 9,925 |
-| [TSV](tsv/wel_latn_nw_phonetic.tsv) | wel | Welsh | Welsh | Latin | North Wales | False | Phonetic | True | 561 |
-| [TSV](tsv/wel_latn_sw_phonetic.tsv) | wel | Welsh | Welsh | Latin | South Wales | False | Phonetic | True | 601 |
-| [TSV](tsv/fry_latn_phonemic.tsv) | fry | Western Frisian | West Frisian | Latin |  | False | Phonemic | True | 977 |
-| [TSV](tsv/apw_latn_phonetic.tsv) | apw | Western Apache | Western Apache | Latin |  | False | Phonetic | True | 158 |
-| [TSV](tsv/mww_latn_phonemic.tsv) | mww | Hmong Daw | White Hmong | Latin |  | False | Phonemic | True | 322 |
-| [TSV](tsv/xho_latn_phonetic.tsv) | xho | Xhosa | Xhosa | Latin |  | False | Phonetic | True | 730 |
-| [TSV](tsv/sah_cyrl_phonemic.tsv) | sah | Yakut | Yakut | Cyrillic |  | False | Phonemic | True | 162 |
-| [TSV](tsv/ybi_deva_phonemic.tsv) | ybi | Yamphu | Yamphu | Devanagari |  | False | Phonemic | False | 112 |
-| [TSV](tsv/yid_hebr_phonemic.tsv) | yid | Yiddish | Yiddish | Hebrew |  | False | Phonemic | False | 251 |
-| [TSV](tsv/yid_hebr_phonetic.tsv) | yid | Yiddish | Yiddish | Hebrew |  | False | Phonetic | False | 358 |
-| [TSV](tsv/zza_latn_phonetic.tsv) | zza | Zaza | Zazaki | Latin |  | False | Phonetic | True | 188 |
-| [TSV](tsv/zha_latn_phonemic.tsv) | zha | Zhuang | Zhuang | Latin |  | False | Phonemic | True | 1,143 |
-| [TSV](tsv/zul_latn_phonemic.tsv) | zul | Zulu | Zulu | Latin |  | False | Phonemic | True | 1,677 |
+(TEMPORARY)
+Manually generated README that links to/describes subdirectories and their README's.
```

Comment:
READMEs? Sorry to bikeshed but the plural apostrophe should be reserved for "A's" and things like that.